build.gradle

```diff
@@ -61,8 +61,8 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```

Comment:
Please also adjust [the README.md](https://github.com/nextcloud/Android-SingleSignOn#1-add-this-library-to-your-project) which still hints to Java 8.

Reply:
@stefan-niedermann If I got it right you only need it if you are using gradle in Version 7.0.0 or higher, right? And it's not backwards compatible with Gradle 4.2, right?